Microsoft.Azure.CosmosRepository/test/Providers/DefaultCosmosContainerProviderTests.cs

```diff
@@ -53,6 +55,7 @@ public class DefaultCosmosContainerProviderTests
                     new TestCosmosClientProvider(),
                     new DefaultCosmosPartitionKeyPathProvider(),
                     null,
+                    new DefaultCosmosUniqueKeyPolicyProvider(),

```

Comment:
We need to add a test case to verify that passing a `null` as the `DefaultCosmosUniqueKeyPolicyProvider` argument throws, like the other tests here.

Reply:
added that in commit bb520e3 ("addes tests")